PcAtChipsetPkg/PcatRealTimeClockRuntimeDxe/PcRtc.c

```diff
@@ -542,6 +556,66 @@ PcRtcGetTime (
   return EFI_SUCCESS;
 }
 
+
+/**
+  Sets the current local time zone information.
+
+  @param  Time                  A pointer to the current time.
+
+  @retval EFI_SUCCESS           The operation completed successfully.
+  @retval EFI_INVALID_PARAMETER A time field is out of range.
+  @retval EFI_DEVICE_ERROR      The time could not be set due due to hardware error.
+
+**/
+EFI_STATUS
+PcRtcSetTimeZone (
+  IN EFI_TIME               *Time
+  )
+{
+  EFI_STATUS      Status;
+  UINT32          TimerVar;
+
+  if (Time == NULL) {
+    return EFI_INVALID_PARAMETER;
+  }
+
+  //
+  // Make sure that the time fields are valid
+  //
+  Status = RtcTimeFieldsValid (Time);
+  if (EFI_ERROR (Status)) {
+    return Status;
+  }

```

Comment:
Can we just use 2 ASSERTs because the function is only called from internal functions?